### PR TITLE
Add podcasts section to community page (id)

### PR DESCRIPTION
**Pull request by meisyal**
As mentioned on PR #1466, changes:

* Add podcasts section to community page
* Add a new page for podcasts.

### CODE REVIEW

**Comment by stomar** [comment]
Maybe let‘s wait a bit for the follow up PR #1999.

**Comment by meisyal** [comment]
Sure @stomar. If there is any changes on that PR, I will also update this translation.

**Comment by meisyal** [comment]
Some commits added because there is changes on #1999.

By the way, #1999 has been merged @stomar.

**Comment by meisyal** [comment]
Hi, there are some changes on this PR (commit b1acdd9 and 76b2d13).

Could you give a review again @gozali or @kuntoaji?

File: id/community/podcasts/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+[Ruby Rogues][rogues]
+: Ruby Rogues podcast adalah sebuah diskusi panel terkait topik yang
+  berhubungan dengan pemrograman, karier, komunitas, dan Ruby.
+
+[Ruby on Rails Podcast][rorpodcast]
+: Ruby on Rails Podcast, sebuah pembicaraan mingguan terkait Ruby on Rails,
+  perangkat lunak *open source*, dan profesi pemrograman.
+
+### Terlibat
```
Comment (kuntoaji) [inline id/community/podcasts/index.md]: I think it's better to use word "Ikut Berkontribusi" or "Kontribusi"

Comment (meisyal) [reply id/community/podcasts/index.md]: Ah, that's a good suggestion. I prefer "Ikut Berkontribusi". @kuntoaji 

Just updated.

**Comment by meisyal** [comment]
Hi @kuntoaji, do you still have a comment for this translation?

Thank you.

**Comment by kuntoaji** [comment]
nope, LGTM..

**Pull request merged**